Packs/qualys/Integrations/integration-qualys.yml

```diff
@@ -1871,9 +1871,7 @@ script:
             - mht
             - xml
             - csv
-          description: 'One output format may be specified. '
-        - name: hide_header
-          description: 0,1
```

Comment:
Why are you changing V1?

Reply:
Yaakovi asked me to add this change in this PR.